### PR TITLE
Specify bool in consul_kv: validate_certs param

### PR DESCRIPTION
**Pull request by jolexa**
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
clustering/consul_kv

##### ANSIBLE VERSION
```
$ ansible --version
ansible 2.1.2.0
  config file =
  configured module search path = Default w/o overrides
```

##### SUMMARY

consul_kv will not connect to our consul cluster

```
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "invocation": {"module_args": {"cas": null, "flags": null, "host": "consul.example.com", "key": "test-jolexa", "port": 443, "recurse": null, "retrieve": "True", "scheme": "https", "session": "session", "state": "present", "token": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER", "validate_certs": "True", "value": "ansible-test"}, "module_name": "consul_kv"}, "msg": "Could not connect to consul agent at consul.svc.spsdev.in:443, error was [Errno 2] No such file or directory"}
```
```
ok: [localhost] => {"changed": false, "data": {"CreateIndex": 202090, "Flags": 0, "Key": "test-jolexa", "LockIndex": 0, "ModifyIndex": 202090, "Value": "ansible-test"}, "index": "202090", "invocation": {"module_args": {"cas": null, "flags": null, "host": "consul.example.com", "key": "test-jolexa", "port": 443, "recurse": null, "retrieve": "True", "scheme": "https", "session": "session", "state": "present", "token": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER", "validate_certs": true, "value": "ansible-test"}, "module_name": "consul_kv"}, "key": "test-jolexa"}
```

As in the original commit, consul.py received bool support but not consul_kv.py. This commit changes validate_certs param to be a bool, instead of a string.

### CODE REVIEW

**Comment by resmo** [comment]
LGTM, could you do the same for `retrieve` as well?

**Comment by gregdek** [comment]
Thanks again to @jolexa for this PR. This PR was reviewed by an Ansible member. Marking for inclusion.

However, we would still appreciate if the maintainer(s) @sgargan could look over it.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by jolexa** [comment]
@resmo Thanks for looking, added a commit and rebased

File: clustering/consul_kv.py [hunk 1]
```diff
@@ -260,7 +260,7 @@ def main():
         key=dict(required=True),
         host=dict(default='localhost'),
         scheme=dict(required=False, default='http'),
-        validate_certs=dict(required=False, default=True),
+        validate_certs=dict(required=False, type='bool', default=True),
         port=dict(default=8500, type='int'),
         recurse=dict(required=False, type='bool'),
         retrieve=dict(required=False, default=True),
```
Comment (resmo) [inline clustering/consul_kv.py]: could you add a type for `retrieve` as well?

**Comment by jolexa** [comment]
Ah, ok. I didn't see that before because it isn't in the docs. Done now.

**Pull request merged**